specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/examples/DeleteRoleDefinition-example.json

```diff
@@ -0,0 +1,30 @@
+{
+    "parameters": {
+        "vaultBaseUrl": "https://myvault.vault.azure.net/",
+        "scope": "",
+        "api-version": "7.2-preview",
```

Comment:
You should re-record these to make sure all data is right. This is for 7.2-preview, which you're putting in the 7.3-preview folder.